### ISSUE TITLE
Public Player Starting Before Fully Loaded

### ISSUE THREAD

**Issue by jackrugile** [rezoner/chirp#14]
The song starts playing before all of the instruments are loaded on the public player. Noticed it here: http://chirp.rezoner.net/player/522cd0056cc1017f3d00000b

**Comment by kirbysayshi** [comment]
Yes, I noticed this too. It manifests as the song stuttering or pitch bending while the ads and social buttons load in on the page.
